app/controllers/TransferAgreementController.scala

```diff
@@ -38,57 +37,65 @@ class TransferAgreementController @Inject()(val controllerComponents: SecurityCo
   )
+
   private val options: Seq[(String, String)] = Seq("Yes" -> "true", "No" -> "false")
   implicit val language: Lang = langs.availables.head
 
-  private def loadPageBasedOnTaStatus(consignmentId: UUID, httpStatus: Status, taForm: Form[TransferAgreementData])
-                                     (implicit request: Request[AnyContent]): Future[Result] = {
+  private def loadStandardPageBasedOnTaStatus(consignmentId: UUID, httpStatus: Status, taForm: Form[TransferAgreementData]=emptyTransferAgreementForm)
+                                        (implicit request: Request[AnyContent]): Future[Result] = {
     val consignmentStatusService = new ConsignmentStatusService(graphqlConfiguration)
 
     consignmentStatusService.consignmentStatus(consignmentId, request.token.bearerAccessToken).map {
       consignmentStatus =>
         val transferAgreementStatus: Option[String] = consignmentStatus.flatMap(_.transferAgreement)
+        val warningMessage = Messages("transferAgreement.warning")
         transferAgreementStatus match {
-          case Some("Completed") => Ok(views.html.standard.transferAgreementAlreadyConfirmed(consignmentId, transferAgreementForm, options)).uncache()
-          case _ =>  httpStatus(views.html.standard.transferAgreement(consignmentId, taForm, options)).uncache()
+          case Some("Completed") =>
+            Ok(views.html.standard.transferAgreementAlreadyConfirmed(consignmentId, emptyTransferAgreementForm, options, warningMessage)).uncache()
+          case _ => httpStatus(views.html.standard.transferAgreement(consignmentId, taForm, options, warningMessage)).uncache()
         }
     }
   }
 
   def transferAgreement(consignmentId: UUID): Action[AnyContent] = secureAction.async { implicit request: Request[AnyContent] =>
-    loadPageBasedOnTaStatus(consignmentId, Ok, transferAgreementForm)
+      loadStandardPageBasedOnTaStatus(consignmentId, Ok)
   }
 
-  def transferAgreementSubmit(consignmentId: UUID): Action[AnyContent] = secureAction.async { implicit request: Request[AnyContent] =>
-    val errorFunction: Form[TransferAgreementData] => Future[Result] = { formWithErrors: Form[TransferAgreementData] =>
-      loadPageBasedOnTaStatus(consignmentId, BadRequest, formWithErrors)
-    }
+  def judgmentTransferAgreement(consignmentId: UUID): Action[AnyContent] = secureAction.async { implicit request: Request[AnyContent] =>
+      val warningMessage = Messages("judgmentTransferAgreement.warning")
+      Future(Ok(views.html.judgment.judgmentTransferAgreement(consignmentId, warningMessage)).uncache())
+  }
 
-    val successFunction: TransferAgreementData => Future[Result] = { formData: TransferAgreementData =>
-      val consignmentStatusService = new ConsignmentStatusService(graphqlConfiguration)
+  def transferAgreementSubmit(consignmentId: UUID): Action[AnyContent] = secureAction.async { implicit request: Request[AnyContent] =>
+    val isJudgmentUser = request.token.isJudgmentUser
```

Comment:
Only calling this once so don't need to define as a val. Just add directly to the if statement: `if (request.token.isJudgmentUser)`